item-detail.html

```diff
@@ -421,8 +421,134 @@
       </div>
     </div>
 
+    <!-- 埋め込みJSでは、兄弟要素を書いたときにエラー時の処理で兄弟が消えるケースがあるので、div でくくります。 Diffが見やすいようにインデントしていません-->
+    <!-- cms-content-typeには実際のコンテンツタイプのIDを指定します-->
+    <div>
+    <div class="p-product-body" cms-item cms-content-type="color-me-items" style="visibility: hidden;">
+      <div class="p-product-body-inner">
+        <{if $product.model != ""}>
+        <div class="p-product-body__model">
+          <{$product.model}>
+        </div>
+        <{/if}>
+
+        <div class="p-product-body__name">
+          <!-- Spearlyの埋め込みコードに置き換え -->
+          {%= color-me-items_title %}
```

Comment:
```suggestion
          {%= sample-ab-testing_title %}
```
こちらも同じくコンテンツタイプIDの違いですね 🙇